lib/tasks/data.rake

```diff
@@ -0,0 +1,29 @@
+namespace :data do
+  desc 'Nullify wrong foreign keys'
+
+  task events: :environment do
```

Comment:
:nullify_nonexistent_foreign_keys sounds good. Then we can expand it to handle other models in the future :-)